.travis.yml

```diff
@@ -9,4 +9,4 @@ node_js:
   - "stable"
 
 script:
-  - npm run lint+build+test
+  - npm run build+test+lint
```

Comment:
It is running them in sequential. I just make them into one command so I can execute them. In newer repo I call it `npm run verify` so I can run it from console.